src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/ImportEmailTemplatesActivity/Global/OrderCreated/Contents/Html/en-CA.html

```diff
@@ -291,6 +291,15 @@ <h2 style="font-size: 24px; margin-bottom: 0px;">
 											</tr>
 											<!--{% endif %}
 										{% endfor %}-->
+										<tr>
```

Comment:
we should also only display that section if we have an adjustment to display (i.e.: don't display it if it is 0$)